src/editor/nodes/collapsibleBlock/nodes/CollapsibleBlockContent.ts

```diff
@@ -7,15 +7,11 @@ export const CollapsibleBlockContentNode = Node.create({
   defining: true,
   selectable: false,
 
-  parseHTML() {
-    return [
-      {
-        tag: 'collapsible-content',
-      },
-    ];
-  },
+  parseHTML: () => [
```

Comment:
I don't mind the left-hand side of this diff, but using arrow functions does allow you to drop the `return` for the common case of a one-liner.